### ISSUE TITLE
Bump snowplow-scala-tracker to 0.6.1

### ISSUE THREAD

**Issue by chuwy** [snowplow/snowplow-elasticsearch-loader#149]
0.5.0 has an incompatibility with iglu core.

**Comment by chuwy** [comment]
Closing in favor of https://github.com/snowplow/snowplow-elasticsearch-loader/issues/131, as Tracker 0.5.0 was the last lib depending on json4s
